src/specs/donation/readme.md

```diff
@@ -0,0 +1,35 @@
+# Aztec Connect Bridges Specification Template
+
+## What does this bridge do? Why did you build it?
+
+The bridge allows people to do a "group-withdraw" that essentially can be seen as donations to an address on L1, from a group of people within Aztec.
+
+## What protocol(s) does the bridge interact with?
+
+The bridge only interacts with the ERC20 tokens that it are to transfer
+
+## What is the flow of the bridge?
+
+The bridge only support one flow, namely sending funds to the specified address. The bridge has an internal mapping of `id`s to addresses, and use the `auxData` to specify the id.
+
+## Please list any edge cases that may restrict the usefulness of the bridge or that the bridge prevents explicit.
+
+The bridge are forwarding 30K gas to the receipient receive function if transferring eth, so if that is insufficient, it will revert.
+
+## How can the accounting of the bridge be impacted by interactions performed by other parties than the bridge? Example, if borrowing, how does it handle liquidations etc.
+
+The bridge can only be impacted by other parties, if a donation is made to a unlisted id.
+If this is the case, someone can list donees until they hit the id and get the donation themselves.
```

Comment:
This seems to be incorrect given that you are checking if receiver is 0 address in the beginning of `convert(...)`.

Reply:
Got it now. Didn't consider that you are talking about a case where the interaction is not yet settled. I would mention this in a spec.